src/components/vm/consoles/VncConsole.jsx

```diff
@@ -56,7 +56,7 @@ export const VncConsole = ({
     vncLogging = 'warn',
     consoleContainerId,
     onConnected = (element) => {},
-    onDisconnected = () => {},
+    onDisconnected = (event) => {},
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2038-86db6651-20250808-063524-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)